.travis/osx..install.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-PACKAGES="cmake pkg-config fftw libogg libvorbis libsndfile libsamplerate jack sdl stk portaudio node fltk"
+PACKAGES="cmake pkgconfig fftw libogg libvorbis lame libsndfile libsamplerate jack sdl libgig libsoundio stk portaudio node fltk"
```

Comment:
> If I change pkgconfig to pkg-config, Travis-CI succeeds. Please check and fix this.

I think this is a coincidence.  Travis is erroring on `shell_session_update: command not found` which seems to be a [ruby issue](https://github.com/direnv/direnv/issues/210#issuecomment-269259123).  We can either merge with Mac failing, or try the recommended fix.